pkg/api/api.go

```diff
@@ -65,7 +70,7 @@ func loggingMiddleware(next http.Handler) http.Handler {
 	})
 }
 
-func NewRouter() *mux.Router {
+func NewRouter(loki lokiClient) *mux.Router {
```

Comment:
This shoudl receive a pointer?

Reply:
even if lokiClient in this file is an interface?

Reply:
ahhhh I didn't notice.

Reply:
okok nvm

Reply:
should I name this `lokiClienter`?

Reply:
Its ok not to end one-method interface names with `er`.

You can keep it like that.